### DISCUSSION TITLE
Recent App, Site, or API Change Disrupts iOS Notifications Feed and Issue/PR Thread Subscription Behavior

### DISCUSSION THREAD

**Discussion by RandomDSdevel** [community/community#13730] [Mobile]
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;A recent app, site, or API change&thinsp;—&thinsp;though, unfortunately, I wouldn't know which one&thinsp;—&thinsp;appears to have disrupted how new issue, PR, and other threads and notices from repositories you're watching are handled when navigating through them from the app notifications feed.  

1. A notification of a new issue or PR or an update to its thread (or some other repository notification) would arrive in your notifications feed from a repository you're watching.  
1. You'd click through the notification to the target page it links to and:  
    - Previously:  
       - You'd, as expected, have been correctly subscribed to that thread automatically.  
    - Now:  
       - You aren't correctly subscribed to the thread automatically to the thread as expected.  

(Note that this is only the case in the app; automatic watched-repository thread subscriptions still appear to work correctly via GitHub's Web interface.)  

---

_Addending edit:_  

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Similarly, iOS app users may now get unsubscribed from closed issue and PR threads automatically when they may still wish to manage their thread subscriptions manually; this should be a per-user/-app instance setting.

**Comment by jacortinas** [comment]
Hello, an issue has been created to research and fix the current subscription implementation if something has broken. Thank you for your feedback, @RandomDSdevel. 🙇

**Reply by RandomDSdevel** [reply]
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Thanks, and you're welcome!

**Comment by RandomDSdevel** [comment]
(Belatedly; apologies:)  

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The release notes for v1.64.0 of GitHub's iOS app noted the following:  

> ⁝
> 
> Bug fixes:  
> ⁝
>  - Repository subscription button bell icon now matches the icon on GitHub.com

This only appears to have been fixed in the top-level notifications view and not when/after you tap through to the issue or PR itself, however.  The 'subscribe'/'unsubscribe' button in the issue or PR's title banner action menu (denoted by an ellipsis) still doesn't reflect issue or PR subscription status in the case where you've set that to get handled automatically by subscribing to notifications of activity on its parent repository.
